src/compiler/emitter.ts

```diff
@@ -446,7 +446,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
             /** If removeComments is true, no leading-comments needed to be emitted **/
             let emitLeadingCommentsOfPosition = compilerOptions.removeComments ? function (pos: number) { } : emitLeadingCommentsOfPositionWorker;
 
-            let moduleEmitDelegates: Map<(node: SourceFile) => void> = {
+            let moduleEmitDelegates: Map<(node: SourceFile, resolvePath?: boolean) => void> = {

```

Comment:
it took me a while to figure out what resolvePath realy does, and i am not sure what is a better name here.